packages/algoliasearch-helper/documentation-src/content/js/main.js

```diff
@@ -110,6 +110,7 @@ function activeLinks(sidebarContainer) {
 
 function getPositionsKeyElements(sidebarContainer) {
   const sidebarBBox = sidebarContainer.getBoundingClientRect();
+  // eslint-disable-next-line compat/compat
```

Comment:
Doesn't work in with Firefox < 59 but I guess it's fine

Reply:
this is the documentation site, can we disable it from this lint rule instead?